spec/lib/search_spec.rb

```diff
@@ -198,6 +198,13 @@
         end
 
         it 'should find only the second page' do
+          # fix the first call for ActiveRecord 7.0
+          # result.first would lead to a find on the id - column which is not present in the order - statement,
+          # which than lead to an Postgres - Grouping error
+          #
+          # For now I leave it that way to fix the test and my future self should find a better solution, if this
+          # is really an issue.
```

Comment:
We could try to use

```rb
result.take
```

This returns the first result without a particular order. Returning what the database thinks is first.